neuromation/cli/formatters/config.py

```diff
@@ -1,35 +1,66 @@
 from sys import platform
+from typing import Dict
 
 from click import style
+from tabulate import tabulate
 
+from neuromation.api.login import RunPreset
 from neuromation.cli.root import Root
 
 
 class ConfigFormatter:
     def __call__(self, root: Root) -> str:
-        if platform == "win32":
-            yes, no = "Yes", "No"
-        else:
-            yes, no = "✔︎", "✖︎"
         lines = []
         lines.append(style("User Name", bold=True) + f": {root.username}")
         lines.append(style("API URL", bold=True) + f": {root.url}")
         lines.append(style("Docker Registry URL", bold=True) + f": {root.registry_url}")
         lines.append(style("Resource Presets", bold=True) + f":")
-        indent = "  "
-        lines.append(f"{indent}Name         #CPU  Memory Preemptible #GPU  GPU Model")
-        for name, preset in root.resource_presets.items():
-            lines.append(
-                (
-                    f"{indent}{name:12}  {preset.cpu:>3} {preset.memory_mb:>7} "
-                    f"{yes if preset.is_preemptible else no:^11}"
-                    f"  {preset.gpu or '':>3}"
-                    f"  {preset.gpu_model or ''}"
-                ).rstrip()
-            )
+        indent: str = "  "
         return (
             style("User Configuration", bold=True)
             + ":\n"
             + indent
             + f"\n{indent}".join(lines)
+            + "\n"
+            + self._format_presets(root.resource_presets)
```

Comment:
indent will be removed, in other case presets table with TPU column can be wider than 80 chars